src/uct/ib/rc/base/rc_iface.h

```diff
@@ -297,6 +295,8 @@ struct uct_rc_iface {
         unsigned             exp_backoff;
         size_t               max_get_zcopy;
 
+        uint32_t             ece;
```

Comment:
i think this change would increase struct size because it's between 2 64-bit fields..

Reply:
@yosefe I'll add it after ```exp_backoff```. The test shows that it won't increase data space after ```exp_backoff```

Reply:
Done